examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1550,6 +1602,12 @@ class GLTFWriter {
 
 			if ( accessor !== null ) {
 
+				if ( isValidVertexAttribute ) {
```

Comment:
nit: do you mind removing the new `isValidVertexAttribute` variable, and instead testing `attributeName.startsWith( '_' )` here? A bit more concise, and I kind of regret our using the word "valid", as custom attributes are still valid.

Reply:
added an extra variable here to keep the readability and the intension of the code simple at this point (it's not immediately obvious why the name is checked for a _ prefix here), but i'm fine with the adjustment too - so it is one variable less 😉